src/index.html

```diff
@@ -10,13 +10,108 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>MOYO</title>
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div class="navbar">
+        <a
+          href="#"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav>
+          <ul class="nav-list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
```

Comment:
The `href` attribute is set to `#`, which is a placeholder. Please replace it with a valid URL to ensure proper navigation.